### ISSUE TITLE
graphene fails to build (meson)

### ISSUE THREAD

**Issue by jbicha** [ebassi/graphene#107]
graphene 1.6.0
meson 0.41.1
glib 2.53.2
gobject-introspection 1.53.2
Ubuntu 17.10 Alpha

The meson build of graphene is no longer working but it built successfully previously.

```
Installing /<<PKGBUILDDIR>>/obj-x86_64-linux-gnu/src/graphene-gobject-1.0.pc to
 /<<PKGBUILDDIR>>/debian/tmp/usr/lib/x86_64-linux-gnu/pkgconfig
Building documentation for graphene

Error in gtkdoc helper script:
'gtkdoc-scangobj' failed with status 127
./graphene-scan: error while loading shared libraries:
 libgraphene-1.0.so.0: cannot open shared object file: No such file or directory
Scan failed: 
```

[Full build log](http://autopkgtest.ubuntu.com/packages/g/graphene/artful/amd64)

**Comment by ebassi** [comment]
This seems like a Meson issue.

**Comment by ebassi** [comment]
Mmh, strange. I can't reproduce locally.

Which version of gtk-doc are you using?

**Comment by jbicha** [comment]
gtk-doc 1.25-7ubuntu1 (There hasn't been a new gtk-doc release in a while.)

**Comment by ebassi** [comment]
It's also 1.6.0, so it may be missing some of the commits that went into master. I'll try and reproduce locally.

**Comment by jbicha** [comment]
meson 0.41 broke stuff so my first guess is meson too.

**Comment by jbicha** [comment]
Laney says that the build works with meson master

**Comment by ebassi** [comment]
I'll probably bump up the dependency in the `graphene-1-6` branch, and spin a new release.

**Comment by jbicha** [comment]
This was fixed by using meson 0.41.1 and cherry-picking this commit:
https://github.com/mesonbuild/meson/pull/1966/commits/6bc14424b

Feel free to close this issue if you like.

**Comment by ebassi** [comment]
Graphene now requires Meson 0.43.1, so we can close this.
